<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_widget</class>
 <widget class="QWidget" name="update_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="updateTabs">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pioneer">
      <attribute name="title">
       <string>Pioneers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableView_pioneers_2">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>First name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Gender</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year born</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year died</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="updateOptionsPioneers">
           <item>
            <property name="text">
             <string>Choose what to update</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>First name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gender</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Year born</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Year died</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="replacePioneersInfo">
           <property name="text">
            <string>Replace with...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="updatePioneer">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateAnotherPioneer">
           <property name="text">
            <string>Continue updating</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="computers">
      <attribute name="title">
       <string>Computers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableView_computers_2">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Computer name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Computer type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year invented</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Was it built?</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="updateOptionsComputers">
         <item>
          <property name="text">
           <string>Choose what to update</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Computer name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Computer type</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Year invented</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Was it built?</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="replaceComputersInfo">
         <property name="text">
          <string>Replace with...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateComputer">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateAnotherComputer">
         <property name="text">
          <string>Continu updating</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="connections">
      <attribute name="title">
       <string>Connections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableView_connections_2">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>First name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Computer</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="updateOptionsConnections">
         <item>
          <property name="text">
           <string>Choose what to update</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pioneer first name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pioneer last name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Computer name</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="replaceConnectionsInfo">
         <property name="text">
          <string>Replace with...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateConnection">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateAnotherConnection">
         <property name="text">
          <string>Continue updating</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
